docs/docs/cmd/planner/task/task-details-get.md

```diff
@@ -0,0 +1,24 @@
+# planner task details get
+
+Retrieve the details of the specified planner task
+
+## Usage
+
+```sh
+m365 planner task get [options]
+```
+
+## Options
+
+`--id <id>`
```

Comment:
Since we renamed the command and the last noun is now `details`, this option should be named `taskId`